_posts/2018-11-11-fs.md

```diff
@@ -0,0 +1,76 @@
+---
+layout: post
+title: The Choices We Make
```

Comment:
I think the title needs to be more descriptive, the title sounds like a best-selling self motivation book.. :)